pkg/ruleguard.rules.go

```diff
@@ -0,0 +1,488 @@
+// +build ignore
+
+package gorules
```

Comment:
Are the commented out code something we've done or from the source?

Are we properly crediting the source of this rule file?

Reply:
@sakjur Should we add a comment documenting the origin of the file?

Reply:
Added a comment crediting origin.